Formula/igv.rb

```diff
@@ -15,7 +15,11 @@ def install
   end
 
   test do
-    (testpath/"script").write "exit"
-    assert_match "Version", shell_output("#{bin}/igv -b script")
+    assert_match "org/broad/igv/ui/IGV.class", shell_output("jar tf #{libexec}/lib/igv.jar")
+    # Fails on Jenkins with Unhandled exception: java.awt.HeadlessException
+    unless ENV["CI"]
+      (testpath/"script").write "exit"
+      assert_match "Version", shell_output("#{bin}/igv -b script")
+    end
```

Comment:
@fxcoudert Any thoughts on this hack? The previous test is failing on Jenkins with `Unhandled exception: java.awt.HeadlessException`. The test succeeds on my local machine. I've disabled this test on CI, and added a fallback test.